### PR TITLE
Sends emails when rent is created

### PR DESCRIPTION
**Pull request by Drocheg**
-Adds .env with environment variables
-Adds secrets from config/secrets.yml as environment variables
-Adds mail trap configuration
-Adds mailer to send the email when rent is created
-Adds sidekiq gem
-Adds dotenv gem

### CODE REVIEW

**Comment by Drocheg** [comment]
https://trello.com/c/MltS60dh/11-mandar-email-al-usuario-cuando-se-crea-un-rent

File: app/controllers/api/v1/rents_controller.rb [hunk 1]
```diff
@@ -13,12 +13,15 @@ def create
         @rent = Rent.new(create_params_sanitized)
 
         if @rent.save
+          RentMailer.new_rent_notification(@rent.id).deliver_later
```
Comment (Gmarn) [inline app/controllers/api/v1/rents_controller.rb]: I recently realize, the `@` here is useless here for define the object rent. Any doubt on why ask me.

File: config/secrets.yml [hunk 2]
```diff
@@ -19,6 +19,14 @@
 
 development:
   secret_key_base: aca23d1befc693835c554a57e4b980ca3cd1d74ea145e07fea6357e1038c672e748e4201bf427ef889053eb56436a76e145dc0167dd17ca55c6a841097d9cb9a
+  mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
```
Comment (Gmarn) [inline config/secrets.yml]: Here, it is not important but do not forgot to add this at least at you production environment. You can use deftault env or import ones from dev in prod

File: app/views/rent_mailer/new_rent_notification.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1>R</h1>
```
Comment (Gmarn) [inline app/views/rent_mailer/new_rent_notification.html.erb]: You can delete this file if you are not using it.

**Review by Gmarn — CHANGES_REQUESTED** [review]

**Review by Gmarn — APPROVED** [review]

**Pull request merged**